coordinator/core/src/test/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/GlobalAggregationCalculatorTest.kt

```diff
@@ -750,6 +731,19 @@ class GlobalAggregationCalculatorTest {
       val expectedAggregations: List<BlobsToAggregate>
     )
 
+    private fun assertOneToAggregationSizeUsesAggregationSize(
+      endSize:
+        UInt,
+      maxAggregationSize: UInt
+    ) {
+      for (aggregationSize in 1u..endSize) {
+        assertThat(
+          GlobalAggregationCalculator.getUpdatedAggregationSize
+          (aggregationSize, maxAggregationSize)
```

Comment:
This formatting looks off. Can we move argument list to the previous line?

Reply:
done - I had just used the base spotlessApply task.